lib/tasks/after_tasks.rake

```diff
@@ -517,5 +517,24 @@ namespace :After do
 
     puts "Finished reindexing tags on hidden and unrevealed works"
   end
+
+  desc "Convert user kudos from users with the official role to guest kudos"
+  task(convert_official_kudos: :environment) do
+    official_users = Role.find_by(name: "official")&.users
+    if official_users.blank?
+      puts "No official users found"
+    else
+      official_users.each do |user|
+        kudos = user.kudos
+        next if kudos.blank?
+
+        puts "Updating #{kudos.size} kudos from #{user.login}"
```

Comment:
If we don't care about 100% accurate or informative output, I can also just do this even if the user has no kudos:
```
        puts "Updating kudos from #{user.login}"
        user.remove_user_from_kudos
      end
```

Reply:
Let's keep the count. It'll be fun to see which official user has the most stray clicks.